lms/templates/instructor/instructor_dashboard_2/metrics.html

```diff
@@ -146,6 +146,30 @@
         metrics_overlay.find('.metrics-student-grades').show();
         metrics_overlay.show();
       });
+      
+      // Click handler for x axis major ticks
+      $('.metrics-container').on("click", '.stacked-bar-graph-axis .tick.major', function () {
+
+        var stackedBarElement;
+        var xValue;
+        var moduleId;
+        var tickIdentifier = d3.select(this).text();
+
+        var stackedBars = $(this).parent().siblings();
+        for (var sbIndex in stackedBars) {
+          var stackedBarElement = stackedBars[sbIndex];
+          if (stackedBarElement.getAttribute("class") == 'stacked-bar') {
```

Comment:
This switches between single and double quotes;
can we consistently use single quotes for non-interpolated strings?